creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -147,6 +148,8 @@ class SnapbackSM {
     await this.stateMachineQueue.add({ startTime: Date.now() })
 
     this.log(`SnapbackSM initialized in ${this.snapbackDevModeEnabled ? 'dev' : 'production'} mode. Added initial stateMachineQueue job; next job in ${stateMachineJobInterval}ms`)
+
+    await SecondarySyncHealthTracker.getAllSyncMetrics()
```

Comment:
is this used anywhere?

Reply:
currently none of these are used, bc i wanted to (1) get the implementation correct and (2) have vicky's reconfig stuff in before consuming these in snapback. wdyt?

Reply:
yes makes total sense